src/main/scala/cromwell/engine/backend/local/LocalEngineFunctions.scala

```diff
@@ -134,6 +135,26 @@ class LocalEngineFunctions(cwd: Path, stdout: Path, stderr: Path) extends LocalE
     } yield file
   }
 
+  override protected def glob(params: Seq[Try[WdlValue]]): Try[WdlArray] = {
+    for {
+      singleArgument <- extractSingleArgument(params)
+      if singleArgument.wdlType == WdlFileType || singleArgument.wdlType == WdlStringType
+      files <- filesMatchingGlob(singleArgument.valueString)
+    } yield WdlArray(WdlArrayType(WdlFileType), files)
+  }
+
+  private def filesMatchingGlob(glob: String): Try[Seq[WdlValue]] = {
+    // The !! process can throw exceptions so do this as a Try:
+    Try {
+      // Weird syntax time! The !! means "run what preceded this as a command and return stdout if the exit code was 0", obviously:
+      val files = Seq("/bin/bash", "-c", s"ls ${new File("").getAbsolutePath}/${cwd.toString}/$glob") !!
```

Comment:
I believe our standards say the interpolation in `s"..."` strings should be kept more minimal than this